### PR TITLE
Prepare release 0.21

### PR DESCRIPTION
**Pull request by SergioEstevao**

### CODE REVIEW

**Comment by SergioEstevao** [comment]
@frosty do you mind to have a look?

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +12,16 @@ All notable changes to this project will be documented in this file.
 - `0.15` Releases - [0.15](#15)
 
 ---
+## [0.20](https://github.com/wordpress-mobile/MediaPicker-iOS/releases/tag/0.21)
```
Comment (frosty) [inline CHANGELOG.md]: `0.20` -> `0.21`

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +12,16 @@ All notable changes to this project will be documented in this file.
...
+Released on 2017-09-06. All issues associated with this milestone can be found using this
+[filter](https://github.com/wordpress-mobile/MediaPicker-iOS/pulls?utf8=✓&q=is%3Apr%20is%3Aclosed%20milestone%3A0.21).
+
+### Fixed
+- Some crashes and bugs: #219 #221 .
+- Fixed bugs relate to selection of assets and refresh:#225 #223 .
```
Comment (frosty) [inline CHANGELOG.md]: Typo and spacing: 

`- Fixed bugs relate to selection of assets and refresh. #225 #223`

In fact, can we update the formatting of these lines to match the ones from the previous version? So:

`- Summary: #123 .` => `- Summary. #123`

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +12,16 @@ All notable changes to this project will be documented in this file.
...
+- Improved performance when capturing new media inside the picker: #211 .
+- Photos captured using the picker where not saving metadata: #226 .
```
Comment (frosty) [inline CHANGELOG.md]: `where` -> `were`

**Comment by SergioEstevao** [comment]
@frosty updated the PR, ready for another look.

**Pull request closed without merging**